registration-processor/core-processor/registration-processor-manual-verification-stage/pom.xml

```diff
@@ -33,11 +37,11 @@
 	</dependencyManagement>
 	<dependencies>
 
-<dependency>
-    <groupId>org.apache.httpcomponents</groupId>
-    <artifactId>httpmime</artifactId>
-    <version>4.3.1</version>
-</dependency>
+		<dependency>
```

Comment:
can u pls not add httpmime here. i think same dependency will be added by auth adapter which may create issue. Kindly verify once. If auth adapter is using same dependency then u dont have to add here. There might be versioning issue because of same.

Reply:
I have not removed dependency due to formatting it is shown like that